src/plugins/intel_gna/src/frontend/scale_factor_calc.cpp

```diff
@@ -1310,37 +1304,6 @@ bool ScaleFactorCalculator::ScaleFactorPerLayerGemm(InferenceEngine::GemmLayer*
     quantData->_dst_quant.SetScale(
             quantData->_src_quant.GetScale() * quantData->_weights_quant.GetScale());
 
```

Comment:
in previous method you left warning , but here removed all, is it intentional?

Reply:
No, it will be fixed.